lib/cc/cli/analyze.rb

```diff
@@ -20,6 +20,9 @@ def run
           runner.run
         end
 
+        unless formatter.empty?
+          exit 1
```

Comment:
We also `exit 1` when analysis errored (which we unfortunately chose to call "failed" in `Formatter`). To distinguish those cases maybe we should exit with a different code for this case?